packages/core-ethereum/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hoprnet/hopr-core-ethereum",
-  "version": "1.90.0-next.21",
```

Comment:
shouldn't the version be master's?

Reply:
Actually it is different for the first merge back.
- on first ones, we need to push the 1.90 release version so the pipeline on master can bump to 1.91.-next...
- on subsequent merge-backs, we use the one from `master` containing `...-next`